oasislmf/cmd/model.py

```diff
@@ -520,59 +520,68 @@ def action(self, args):
         source_exposures_validation_file_path = as_path(
             inputs.get('source_exposures_validation_file_path', required=False, is_path=True),
             'Source exposures validation file',
             preexists=False
         )
+        canonical_to_model_exposures_transformation_file_path = as_path(
+            inputs.get('canonical_to_model_exposures_transformation_file_path', required=True, is_path=True),
+            'Canonical to model exposures transformation file path'
+        )
+        canonical_exposures_validation_file_path = as_path(
+            inputs.get('canonical_exposures_validation_file_path', required=False, is_path=True),
+            'Canonical exposures validation file',
+            preexists=False
+        )
+        source_accounts_file_path = as_path(
+            inputs.get('source_accounts_file_path', required=False, is_path=True), 'Source accounts file path',
+            preexists=False
+        )
         source_accounts_validation_file_path = as_path(
             inputs.get('source_accounts_validation_file_path', required=False, is_path=True),
-            'Source accounts file validation file path'
+            'Source accounts file validation file path',
+            preexists=False
         )
-        source_to_canonical_exposures_transformation_file_path = as_path(
-            inputs.get('source_to_canonical_exposures_transformation_file_path', required=True, is_path=True),
-            'Source to canonical exposures file transformation file path'
+        canonical_accounts_profile_json_path = as_path(
+            inputs.get('canonical_accounts_profile_json_path', required=False, is_path=True),
+            'Supplier canonical accounts profile JSON path',
+            preexists=False
         )
         source_to_canonical_accounts_transformation_file_path = as_path(
             inputs.get('source_to_canonical_accounts_transformation_file_path', required=False, is_path=True),
-            'Source to canonical accounts file transformation file path'
-        )
-        canonical_exposures_validation_file_path = as_path(
-            inputs.get('canonical_exposures_validation_file_path', required=False, is_path=True),
-            'Canonical exposures validation file',
+            'Source to canonical accounts file transformation file path',
             preexists=False
         )
-        canonical_to_model_exposures_transformation_file_path = as_path(
-            inputs.get('canonical_to_model_exposures_transformation_file_path', required=True, is_path=True),
-            'Canonical to model exposures transformation file path'
-        )
-
-        fm = inputs.get('fm', default=False)
-
         fm_agg_profile_path = as_path(
             inputs.get('fm_agg_profile_path', required=False, is_path=True),
-            'Supplier FM aggregation profile JSON file path'
+            'Supplier FM aggregation profile JSON file path',
```

Comment:
The aggregation profile is not tied to a supplier - this is just a command help message, but we can remove things like these in subsequent releases.